momentum/changelog/legacy/2-x-legacy-2003-2006.md

```diff
@@ -0,0 +1,561 @@
+---
+title: "Momentum 2.x Legacy Changelog from 2003-2006"
```

Comment:
It looks like there are some Momentum 1.x changelogs here too....Maybe just say "Momentum Legacy Changelogs from 2003-2006"?

Reply:
There may be, but this was what was on support.messagesystems.com, all under 2.x changelog, and I don't think it's worth the effort to rename all of this stuff and try to separate it out. We may not be keeping it. 
@JohnPeacockMessageSystems Do we even want to keep the 2.x, message scope, and message central changelogs on this page? Or should we get rid of them?

Reply:
@JohnPeacockMessageSystems ^ Are there any changelogs on here we should get rid of, or should we leave all of the ones up?